completions/puppet

```diff
@@ -276,6 +281,27 @@ _puppet()
                     return
             esac
             ;;
+	lookup)
+	    case $prev in
+		--facts)
+		    _filedir json
+		    return
+		    ;;
+		--node)
+		    _known_hosts_real -- "$cur"
+		    return
+		    ;;
+		--merge)
+		    _puppet_hash_opts
```

Comment:
Please just inline _puppet_hash_opts here if it's not going to be used from anywhere else.